app/Http/Controllers/Web/GroupTypesController.php

```diff
@@ -66,11 +69,11 @@ public function edit(GroupType $groupType)
      */
     public function update(GroupType $groupType, Request $request)
     {
-        // @TODO: Override unique rule.
-        // @see https://github.com/DoSomething/rogue/pull/1030#discussion_r434628475
-        $this->validate($request, $this->rules);
+        $values = $this->validate($request, array_merge_recursive($this->rules, [
+            'name' => [Rule::unique('group_types')->ignore($groupType->id)],
+        ]));
```

Comment:
I'm confused about why we have to do this here since we're not explicitly pulling in the `'name' => [Rule::unique('group_types')]` here 🤔 (also confused about this in the campaign example linked from the linked comment!)

Reply:
There's a "Forcing A Unique Rule To Ignore A Given ID" section in the docs on the Laravel [unique](https://laravel.com/docs/5.5/validation#rule-unique) rule that I found helpful per the changes requested in https://github.com/DoSomething/rogue/pull/1030#discussion_r434628475:

From the docs example of updating an existing user's profile:

> Of course, you will want to verify that the e-mail address is unique. However, if the user only changes the name field and not the e-mail field, you do not want a validation error to be thrown because the user is already the owner of the e-mail address.

Reply:
Hmm okay that makes sense but I'm still confused because it doesn't seem like that unique rule that we define in the `store` method would get applied here anyway, so I'm confused why we have to ignore the rule. Does that make sense?

Reply:
The `name` column in `group_types` tables has a unique index, does that help clarify? https://github.com/DoSomething/rogue/blob/50b4d5faee4cc393976f29446a45ff9653561304/database/migrations/2020_05_27_000000_create_group_types_table.php#L16-L20

Once we add another editable field to this form, e.g. "Should collect School ID", then we'd run into trouble trying to update a Group Type when not updating the name, but the additional fields.

Reply:
AH yes that does! thank you!